docs/whats-new.md

```diff
@@ -2,6 +2,33 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.7
+
+Release date: February 24, 2022
+
+### MaskExtension
+
+The new [MaskExtension](/docs/api-reference/extensions/mask-extension) allows layers to show/hide objects by a geofence. For example, a map may filter a list of user locations by the boundaries of a given country, or highlight part of a base map that is inside a user-drawn circle or lasso area.
```

Comment:
The documentation page for MaskExtension exists but it is not linked in the table of contents.

Reply:
I fixed this in a separate PR https://github.com/visgl/deck.gl/pull/6680